Configurations/90-team.norelease.conf

```diff
@@ -96,7 +96,7 @@ my %targets = (
         cc               => "clang",
         cflags           => combine("-arch x86_64 -DL_ENDIAN",
                                     join(' ', @gcc_devteam_warn),
-                                    "-Wno-error=overlength-strings -Wno-error=extended-offsetof -Wno-error=language-extension-token -Wno-error=unused-const-variable -Wstrict-overflow -Qunused-arguments -DBN_DEBUG -DCONF_DEBUG -DDEBUG_SAFESTACK -DDEBUG_UNUSED -g3 -O3 -pipe",
+                                    "-Wno-error=overlength-strings -Wno-error=extended-offsetof -Wno-error=language-extension-token -Wno-error=unused-const-variable -Wstrict-overflow -Qunused-arguments -DBN_DEBUG -DCONF_DEBUG -g3 -O3 -pipe",
```

Comment:
You also removed `DEBUG_UNUSED` here, but not in the "debug-test-64-clang" target.  Please either remove it in all places or restore it here.

Reply:
Ups, I did not notice that in the unified diff. Let me check whether that define is obsolete, too. If it is, then I'll remove it, too.

Reply:
It is still used:

https://github.com/openssl/openssl/blob/bf5b04ea25d6ac7d31e388b4e87d3eb5cd1e1e2b/include/openssl/e_os2.h#L224-L228

Reply:
Fixed in f53845b9081837ca4aa669b5318791d0baa4df8f.